pkg/dio/multi.go

```diff
@@ -39,20 +39,26 @@ type multiWriter struct {
 	writers []io.Writer
 }
 
-func (t *multiWriter) Write(p []byte) (n int, err error) {
+func (t *multiWriter) Write(p []byte) (int, error) {
+	var n int
+	var err error
+
 	// stash a local copy of the slice as we never want to write twice to a single writer
 	// if remove is called during this flow
 	wTmp := t.writers
 
 	if verbose {
-		log.Debugf("[%p] writing \"%s\" to %d writers", t, string(p), len(t.writers))
+		defer func() {
```

Comment:
defer a log message in a deferred function in an if-then block.  What's the desired behavior?